api/api/events/models.py

```diff
@@ -0,0 +1,16 @@
+from accounts.models import User
+from clubs.models import Club
+from django.db import models
+
+
+class Events(models.Model):
+    name = models.CharField(max_length=100)
+    description = models.TextField()
+    starts_at = models.DateField()
+    ends_at = models.DateField()
+    registration_required = models.BooleanField(default=False)
+    created_by = models.OneToOneField(User, on_delete=models.PROTECT)
```

Comment:
`created_by` should be set `null` if the user is deleted. For that you'd need to make this field nullable as well.